pkg/apis/core/types.go

```diff
@@ -2686,6 +2686,12 @@ type PodSpec struct {
 	// If not specified, the default is true.
 	// +optional
 	EnableServiceLinks *bool
+	// TopologySpreadConstraints describes how a group of pods ought to spread across topology
```

Comment:
nit: replace `ought to` with `should be`

Reply:
The sentence is grammatical as written, please don't change.